docs/application/dotnet/guides/nui/nui-components/LottieAnimationView.md

```diff
@@ -0,0 +1,55 @@
+# LottieAnimationView
+LottieAnimationView is a common component that is used to show a vector animation. It parses the Adobe After Effects® animations exported as JSON with Bodymovin and renders them.
+
+![LottieAnimationView](./media/lottie.gif)
+
+## Create with property
+
+To create a LottieAnimationView using property, follow these steps:
+
+1. Create LottieAnimationView using the default constructor:
+
+    ```cs
+    lottieAnimationView = new LottieAnimationView();
+    ```
+
+2. Set the properties:
+
+    ```cs
+    lottieAnimationView.Size = new Size(100, 108);
+    lottieAnimationView.URL = "myLottie.json"
+    lottieAnimationView.LoopCount = 3;
+    lottieAnimationView.StopBehavior = StopBehaviorType.CurrentFrame;
+    lottieAnimationView.CurrentFrame = 20;
+
+    lottieAnimationView.SetMinMaxFrame(10, 30);
+
+    parent.Add(lottieAnimationView);
+    ```
+
+## Control animation playback
+After the view is created, you can control its playback:
+
+```cs
+lottieAnimationView.Play();
+lottieAnimationView.Pause();
+lottieAnimationView.Stop();
+```
+
+## Event handling
+You can receive an event when the animation finishes:
+
+```cs
+lottieAnimationView.Finished += OnAnimationFinished;
+```
+
+```cs
+private void OnAnimationFinished(object sender)
+{
+  // Do something
+}
+```
+
+## Related information
+- Dependencies
+  -   Tizen 6.0 and Higher
```

Comment:
Question.

The master branch is to handle the latest published version, Tizen 5.5 now.
If the LottieAnimationView is supported since Tizen 6.0,
it must be targeting tizen_6.0_prepare branch.

Would you please check the version again?

Reply:
Tizen 5.5 is right. I've updated the version.